sources/Items/Bonus.cpp

```diff
@@ -45,12 +43,23 @@ namespace BBM {
 		playerBonus->nextSpeedBonusRate = playerBonus->speedBonusRate;
 	}
 
+	void Bonus::NoClipBonus(WAL::Entity &player, const WAL::Entity &bonus, CollisionComponent::CollidedAxis axis)
+	{
+		if (bonus.shouldDelete() || axis != 7)
```

Comment:
Don't use ints for enum values. Define the value or use the operator|

Reply:
lol